packages/trader/src/Modules/Account/Sections/Security/ConnectedApps/connected-apps.jsx

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { DesktopWrapper, MobileWrapper, Button, Modal, Icon } from '@deriv/components';
+import { localize } from '@deriv/translations';
+import { connect } from 'Stores/connect';
+import ErrorComponent from 'App/Components/Elements/Errors';
+import DataTable from 'App/Components/Elements/DataTable';
+import DataList from 'App/Components/Elements/DataList';
+import getConnectedAppsColumnsTemplate from './data-table-template';
+import Loading from '../../../../../templates/_common/components/loading.jsx';
+
+class ConnectedApps extends React.Component {
+    state = {
+        is_loading: true,
+        is_modal_open: false,
+        selected_app_id: null,
+        is_error: false,
+    };
+
+    componentDidMount() {
+        this.updateApp();
+    }
+
+    handleRevokeAccess = () => {
+        this.setState({ is_modal_open: false, is_loading: true });
+        this.props.revokeAccess(this.state.selected_app_id).then(this.updateApp);
+    };
+
+    handleToggleModal = (app_id = null) => {
+        if (this.state.is_modal_open) {
+            this.setState({ is_modal_open: false, selected_app_id: null });
+        } else {
+            this.setState({ is_modal_open: true, selected_app_id: app_id });
+        }
+    };
+
+    updateApp = () => {
+        this.props
+            .fetchConnectedApps()
+            .then(() => {
+                this.setState({ is_loading: false });
+            })
+            .catch(() => {
+                this.setState({ is_error: true });
+            });
+    };
+
+    columns_map = getConnectedAppsColumnsTemplate(this.handleToggleModal).reduce((map, item) => {
+        map[item.col_index] = item;
+        return map;
+    }, {});
+
+    mobileRowRenderer = ({ row }) => {
+        return (
+            <div className='data-list__row'>
+                <div className='data-list__col'>
+                    <DataList.Cell row={row} column={this.columns_map.name} />
+                    <DataList.Cell row={row} column={this.columns_map.scopes} />
+                </div>
+                <div className='data-list__col--small'>
+                    <DataList.Cell row={row} column={this.columns_map.last_used} />
+                    <DataList.Cell row={row} column={this.columns_map.app_id} is_footer={true} />
+                </div>
+            </div>
+        );
+    };
+
+    render() {
+        return (
+            <section className='connected-apps'>
+                <p className='connected-apps__title'>{localize('Authorised applications')}</p>
+                {this.state.is_error ? (
+                    <ErrorComponent />
+                ) : (
+                    <React.Fragment>
+                        <DesktopWrapper>
+                            <DataTable
```

Comment:
Seems like the table design the same between `Connected apps` and `Login history`, perhaps you could collaborate on this so we don't have two implementations of the same design? And is it possible to use our `Table` component in components? @salar-fs @arash-fs

Reply:
I think we can merge these two together later on a new card.

Reply:
As discussed it's fine 👍